### DISCUSSION TITLE
Enable Telescope `path_display = {"smart"}` by default

### DISCUSSION THREAD

**Discussion by trongthanh** [NvChad/NvChad#3074] [Ideas]
My suggestion is we turn on the Telescope settings `path_display = {"smart"}` by default in `/lua/nvchad/configs/telescope.lua`. I think this is the behavior common in other IDEs that they will truncate long file path and focus on the file names.

See this [Telescope issue](https://github.com/nvim-telescope/telescope.nvim/issues/895) for more info.

**Comment by siduck** [comment]
sorry for late reply, idk how i missed this notif!  Can you show with/without this change? i tried and found no difference
